src/main/java/org/cascadebot/cascadebot/commands/core/StatsCommand.java

```diff
@@ -0,0 +1,49 @@
+package org.cascadebot.cascadebot.commands.core;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import org.apache.commons.io.FileUtils;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandCore;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+
+import com.sun.management.OperatingSystemMXBean;
+
+import java.lang.management.ManagementFactory;
+
+public class StatsCommand implements ICommandCore {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        EmbedBuilder builder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+        OperatingSystemMXBean osBean = ManagementFactory.getPlatformMXBean(OperatingSystemMXBean.class);
+
+        builder.setTitle(context.getSelfUser().getName());
+        builder.setThumbnail(context.getSelfUser().getAvatarUrl());
+
+        builder.addField("Total Guilds", String.valueOf(CascadeBot.INS.getShardManager().getGuilds().size()), true);
+        builder.addField("Active Guilds", String.valueOf(CascadeBot.INS.getShardManager().getGuildCache().size()), true);
+        builder.addField("Active Voice Channels", String.valueOf(CascadeBot.INS.getShardManager().getVoiceChannels().size()), true);
```

Comment:
This needs to be filtered :P

Reply:
I dunno how to do this, could you help me?